build.gradle

```diff
@@ -166,54 +168,64 @@ tasks.register('integrationTest', Test) {
 }
 
 dependencies {
-    api platform("com.exactpro.th2:bom:3.2.0")
-    api ("com.exactpro.th2:cradle-core:${cradleVersion}")
+    api platform("com.exactpro.th2:bom:4.0.0-vulnerabilities-2919290091-SNAPSHOT")
+    api ("com.exactpro.th2:cradle-core:${cradleVersion}") {
+        exclude group: 'org.slf4j', module: 'slf4j-log4j12' // because of the vulnerability
+    }
     api 'com.exactpro.th2:grpc-common:3.11.1'
 
     implementation 'com.google.protobuf:protobuf-java-util'
     implementation 'com.exactpro.th2:grpc-service-generator:3.2.2'
-    implementation ("com.exactpro.th2:cradle-cassandra:${cradleVersion}")
+    implementation ("com.exactpro.th2:cradle-cassandra:${cradleVersion}") {
+        exclude group: 'org.slf4j', module: 'slf4j-log4j12' // because of the vulnerability
+    }
 
     //FIXME: Add these dependencies as api to grpc-... artifacts
     implementation "io.grpc:grpc-protobuf"
     implementation "io.grpc:grpc-core"
     implementation "io.grpc:grpc-netty"
-    implementation "io.grpc:grpc-services:1.32.1" //FIXME: add to bom
+    implementation "io.grpc:grpc-services"
 
     implementation "com.rabbitmq:amqp-client"
 
     implementation "org.jetbrains:annotations"
 
     implementation "org.apache.commons:commons-lang3"
-    implementation "org.apache.commons:commons-collections4:4.4"
+    api("org.apache.commons:commons-collections4:4.4") {
```

Comment:
Maybe add it to bom too

Reply:
If we have added it into the bom you can remove version from here. The same for other dependencies which you have added to the bom